Xamarin.Forms.Platform.UAP/SearchBarRenderer.cs

```diff
@@ -121,6 +122,16 @@ void OnControlLoaded(object sender, RoutedEventArgs routedEventArgs)
 								|| !Element.OnThisPlatform().GetIsLegacyColorModeEnabled();
 		}
 
+		protected override void SetAutomationId(string id)
+		{
+			base.SetAutomationId(id);
+
+			if (_queryTextBox == null)
+				return;
+
+			_queryTextBox.SetAutomationPropertiesAutomationId($"{id}_AutoSuggestBox");
```

Comment:
@samhouts is this alright? I use this to locate the inner text box for automation to make entering text more reliable.